edgelet/edgelet-docker/src/settings.rs

```diff
@@ -33,6 +34,7 @@ pub struct MobyRuntime {
     #[serde(with = "url_serde")]
     uri: Url,
     network: MobyNetwork,
+    content_trust_root_ca_registry: Option<HashMap<String, PathBuf>>,
```

Comment:
Can we shorten this name? In fact, your tests had a short name for this :)  
```suggestion
    content_trusted_certs: Option<HashMap<String, PathBuf>>,
```